.github/workflows/build-and-test.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         run: cmake --build build --target install
 
       - name: Upload built plugin (Linux glibc 2.31)
-        if: matrix.os == 'ubuntu-20.04'
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
This means the binary uploaded for each release will have a different minimum required glibc version (2.35, not 2.31 as stated in the above comment and on the release page https://github.com/norberttak/XPanel/releases/tag/v1.8https://github.com/norberttak/XPanel/releases/tag/v1.8)

Reply:
The original purpose of this PR was to move from the old Ubuntu 20.04 to 22.04 and later (once it is released) to 24.04 that's why I added ubuntu-latest. 
Just to be sure I understand: If I want to support the latest Ubuntu stable version, I need to use ubuntu-22.04 and update the glibc version text from 2.31 to 2.35.